buildpipeline/Core-Setup-Linux-BT.json

```diff
@@ -1043,6 +1206,22 @@
     "CliNuGetApiKey": {
       "value": null,
       "isSecret": true
+    },
+    "DistroRid_Ubuntu1704": {
```

Comment:
@eerhardt changed how these are done recently so make sure you are following the new pattern.

Reply:
[That change](https://github.com/dotnet/core-setup/pull/3291) hasn't been merged yet.  It probably won't get merged until after this change, so don't worry about it.  I'll resolve any conflicts.